gulp.config.js

```diff
@@ -41,34 +45,39 @@ module.exports = function () {
     var report = {
         path: 'report/'
     };
-    var browserSync = {
-        dev: {
-            port: 3000,
-            server: {
-                baseDir: './src/',
-                middleware: [historyApiFallback()],
-                routes: {
-                    "/node_modules": "node_modules",
-                    "/src": "src"
-                }
+
+    if (global.env === 'dev')
```

Comment:
Added 
global.env = 'dev';
global.env = argv.prod ? 'prod': 'dev';

To gulp.file.js